src/components/App/AppIFrame.js

```diff
@@ -45,6 +46,12 @@ const SANDBOX = [
 ].join(' ')
 
 class AppIFrame extends React.Component {
+  static = {
+    app: PropTypes.object.isRequired,
+    ref: PropTypes.func.isRequired,
+    onLoad: PropTypes.func.isRequired,
```

Comment:
Just a note that `onLoad` isn't required

Reply:
Actually surprised this even compiled haha, we missed the `propTypes` declaration here too!